hs/Database/Tables.hs

```diff
@@ -183,29 +162,19 @@ data Course =
 instance ToJSON Course
 instance ToJSON Session
 
-instance ToJSON Lecture where
-  toJSON (Lecture lectureExtra lectureSection lectureCap lectureTimeStr lectureTime lectureInstructor lectureEnrol lectureWait)
-          = object ["extra" .= lectureExtra,
-                    "section" .= lectureSection,
-                    "cap" .= lectureCap,
-                    "time_str" .= lectureTimeStr,
-                    "time" .= map convertTimeToString lectureTime,
-                    "instructor" .= lectureInstructor,
-                    "enrol" .= lectureEnrol,
-                    "wait" .= lectureWait
-                   ]
-
-instance ToJSON Tutorial where
-  toJSON (Tutorial Nothing tutorialTimes tutorialTimeStr) =
-      Array $ V.fromList [toJSON (map convertTimeToString tutorialTimes), toJSON tutorialTimeStr]
-  toJSON (Tutorial (Just value) tutorialTimes tutorialTimeStr) =
-      Array $ V.fromList [toJSON value, toJSON (map convertTimeToString tutorialTimes), toJSON tutorialTimeStr]
+instance ToJSON Time where
+    toJSON (time) =
+        toJSON $ convertTimeToString time
+
+-- instance FromJSON required so that tables can be parsed into JSON,
+-- not necessary otherwise.
+instance FromJSON Time
```

Comment:
I have my own understanding why that wouldn't be the best idea, but I have no idea why you'd rather not. My guess would be that using a String/Text is too general a type to represent the time, and it makes us lose a bit of certainty about what exactly is in the database. However, making changes without clearly knowing the reason why I'm making the changes isn't very good in my opinion. I am now prompted to ask this question: Why would you rather not?